config/opal_dep_version.m4

```diff
@@ -0,0 +1,50 @@
+dnl -*- shell-script -*-
+dnl
+dnl Copyright (c) 2022      IBM Corporation.  All rights reserved.
+dnl
+dnl $COPYRIGHT$
+dnl
+dnl Additional copyrights may follow
+dnl
+dnl $HEADER$
+dnl
+
+# OPAL_GET_DEP_VERSION(version_file, variable_prefix)
+# -----------------------------------------------
+# Parse version_file for dependent version information, setting
+# the following shell variables:
+# OMPI_$2_MIN_VERSION
+# OMPI_$2_NUMERIC_MIN_VERSION
+#
+AC_DEFUN([OPAL_GET_DEP_VERSION],[
+    dnl quote eval to suppress macro expansion with non-GNU m4
+    if test -f "$1"; then
+        dep_vers=`sed -n "
+        t clear
+        : clear
+        s/^$2_min_version/[OMPI_]m4_toupper($2)[_MIN_VERSION]/
+        t print
+        b
+        : print
+        p" < "$1"`
+        [eval] "$dep_vers"
+
+        IFS_SAVE=$IFS
+        IFS="."
+        read -a strarr <<< "${$2_OMPI_MIN_VERSION}"
```

Comment:
We chatted in Slack: this `<<<` might be a bash-ism, and therefore no good.  Also, you have the value in an env variable already; this `read` might not be necessary (because you're just copying it to another env variable).

Reply:
Replaced with `cut`